src/applications/third-party-apps/components/SearchResult/index.jsx

```diff
@@ -0,0 +1,147 @@
+// Node modules.
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import join from 'lodash/join';
+import map from 'lodash/map';
+
+export class SearchResult extends Component {
+  static propTypes = {
+    item: PropTypes.shape({
+      appURL: PropTypes.string.isRequired,
+      categories: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+      description: PropTypes.string.isRequired,
+      iconURL: PropTypes.string.isRequired,
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+      permissions: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+      platforms: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+      privacyPolicyURL: PropTypes.string.isRequired,
+      termsOfServiceURL: PropTypes.string.isRequired,
+    }).isRequired,
+  };
```

Comment:
I think we can do what we did in Find Forms awhile back and centralize this prop type into a `directory/prop-types.js` file so we only need to define the shape one time